### ISSUE TITLE
Consider including more content on data modelling and data dictionary

### ISSUE THREAD

**Issue by vildead** [elixir-luxembourg/DS-DM-training#23]
Danielle had nice presentation on 2022-02-07 covering this topic ;)

**Comment by rolandkrause** [comment]
Agreed. It would require a training on its own. We need to develop and hold it internally first.

**Comment by vildead** [comment]
very nice slides from ELIXIR-NO: https://github.com/ELIXIR-Norway-Training/DMP-writing-workshop/blob/main/07_metadata_standards_ontologies.pdf

**Comment by vildead** [comment]
This is really important - PhD candidates were not aware of data dictionary
